pobieracz_danych_gugik.py

```diff
@@ -167,124 +170,124 @@ def unload(self):
+        self.dockwidget.wizualizacja_karto_fromLayer_btn.clicked.connect(
+            self.wizualizacja_karto_fromLayer_btn_clicked)
 
-            self.dockwidget.osnowa_arch_capture_btn.clicked.connect(
-                lambda: self.capture_btn_clicked(self.kartoteki_osnowClickTool))
-            self.dockwidget.osnowa_arch_fromLayer_btn.clicked.connect(self.osnowa_arch_fromLayer_btn_clicked)
+        self.dockwidget.osnowa_arch_capture_btn.clicked.connect(
+            lambda: self.capture_btn_clicked(self.kartoteki_osnowClickTool))
+        self.dockwidget.osnowa_arch_fromLayer_btn.clicked.connect(self.osnowa_arch_fromLayer_btn_clicked)
 
-            self.dockwidget.archiwalne_bdot_selected_powiat_btn.clicked.connect(
-                self.archiwalne_bdot_selected_powiat_btn_clicked)
+        self.dockwidget.archiwalne_bdot_selected_powiat_btn.clicked.connect(
+            self.archiwalne_bdot_selected_powiat_btn_clicked)
 
-            self.dockwidget.zdjecia_lotnicze_capture_btn.clicked.connect(
-                lambda: self.capture_btn_clicked(self.zdjecia_lotniczeClickTool))
-            self.dockwidget.zdjecia_lotnicze_fromLayer_btn.clicked.connect(self.zdjecia_lotnicze_fromLayer_btn_clicked)
+        self.dockwidget.zdjecia_lotnicze_capture_btn.clicked.connect(
+            lambda: self.capture_btn_clicked(self.zdjecia_lotniczeClickTool))
+        self.dockwidget.zdjecia_lotnicze_fromLayer_btn.clicked.connect(self.zdjecia_lotnicze_fromLayer_btn_clicked)
 
-            # connect to provide cleanup on closing of dockwidget
-            self.dockwidget.closingPlugin.connect(self.onClosePlugin)
+        # connect to provide cleanup on closing of dockwidget
+        self.dockwidget.closingPlugin.connect(self.onClosePlugin)
 
-            self.dockwidget.folder_fileWidget.setFilePath(
-                self.settings.value("pobieracz_danych_gugik/settings/defaultPath", ""))
-            self.dockwidget.folder_fileWidget.fileChanged.connect(
-                lambda: self.settings.setValue("pobieracz_danych_gugik/settings/defaultPath",
-                                               self.dockwidget.folder_fileWidget.filePath()))
+        self.dockwidget.folder_fileWidget.setFilePath(
+            self.settings.value("pobieracz_danych_gugik/settings/defaultPath", ""))
+        self.dockwidget.folder_fileWidget.fileChanged.connect(
+            lambda: self.settings.setValue("pobieracz_danych_gugik/settings/defaultPath",
+                                           self.dockwidget.folder_fileWidget.filePath()))
 
-            # informacje o wersji
-            self.dockwidget.setWindowTitle('%s %s' % (plugin_name, plugin_version))
-            self.dockwidget.lbl_pluginVersion.setText('%s %s' % (plugin_name, plugin_version))
+        # informacje o wersji
+        self.dockwidget.setWindowTitle(f'{plugin_name} {plugin_version}')
+        self.dockwidget.lbl_pluginVersion.setText(f'{plugin_name} {plugin_version}')
```

Comment:
To może lepiej zostaw tak jak było, bo takie zastosowanie mamy we wszystkich innych wtyczkach